opencog/cython/opencog/atomspace.pxd

```diff
@@ -82,7 +82,7 @@ cdef extern from "opencog/atomspace/atom_types.h" namespace "opencog":
     cdef Type NOTYPE
 
 # Handle
-ctypedef long UUID
+ctypedef public long UUID
```

Comment:
You can get a minor performance bump by figuring out how to store the entire Handle, instead of the integer UUID only.  By working with UUID's , you incurr the cost of looking up the actual C++ atom that corresponds to the UUID.